Functions.Templates/Templates/SqlInputBinding-JavaScript/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "type": "sql",
+      "name": "results",
+      "direction": "in",
+      "commandText": "select * from tableName",
```

Comment:
In the readme, we talk about `[dbo].[table1]`. Should we be consistent here, @dzsquared, or does including the schema complicate things too much?

Reply:
My preference is always include the scheme (and quote both objects). It's just good practice and can help avoid potential mistakes so we should be encouraging that where possible.